File: task-03/Go/Sub-Task3.go
package main
import "fmt"

func main() {
    var n int
    fmt.Print("Enter a number: ")
    fmt.Scan(&n)

    for i := 0; i < n; i++ {
        fmt.Println(" " * (n - i - 1) + "*" * (2 * i + 1))
    }
    for i := n-2; i >= 0; i-- {
        fmt.Println(" " * (n - i - 1) + "*" * (2 * i + 1))
    }
}

File: task-03/Rust/Sub-Task3.rs
use std::io;

fn main() {
    let mut input = String::new();
    println!("Enter a number: ");
    io::stdin().read_line(&mut input).expect("Failed to read input");
    let n: i32 = input.trim().parse().expect("Invalid number");

    for i in 0..n {
        println!("{}{}", " ".repeat((n-i-1) as usize), "*".repeat((2*i+1) as usize));
    }

    for i in (0..n-1).rev() {
        println!("{}{}", " ".repeat((n-i-1) as usize), "*".repeat((2*i+1) as usize));
    }
}
